composeml/label_times.py

```diff
@@ -315,28 +333,26 @@ def sample(self, n=None, frac=None, random_state=None):
             return sample
 
         if isinstance(n, dict):
-            sample_per_label = []
+            sample_per_label = self.head(0)
             for label, n, in n.items():
                 label = self[self[self.name] == label]
                 sample = label.sample(n=n, random_state=random_state)
-                sample_per_label.append(sample)
+                sample_per_label = sample_per_label.append(sample)
```

Comment:
now that we have finalize, can we go back to the previous concat approach?

Reply:
Yes, concat and append both work.

Reply:
i think concat is the better way to do it then repeated calls to append, so i guess go back to that?

Reply:
Yes, sounds good.